ORA.Core/Encryption/RSACipher.cs

```diff
@@ -0,0 +1,62 @@
+﻿using ORA.API.Encryption;
+using System.Security.Cryptography;
+
+namespace ORA.Core.Encryption
+{
+    public class RsaCipher : ICipher
+    {
+        private RSAParameters publickey;
+        private RSAParameters privatekey;
+
+        public RsaCipher()
```

Comment:
Could be a great idea to let the choice of the size of the keys in the constructor

Reply:
@TrAyZeN ?

Reply:
We will be using only one size of key, but if it makes sense to you that people who want to build something on top of ORA want to change the key size then yes this is a good idea.